frame/module_bdywrite.F

```diff
@@ -143,7 +143,7 @@ subroutine bdywrite(grid,filename)
     !        ims,ime, jms,jme, kms,kme, &
     !        ips,ipe, jps,jpe, kps,kpe)
 
-    assert(nf_create(trim(filename),NF_64BIT_OFFSET,fid))
+    assert(nf_create(trim(filename),NF_64BIT_DATA,fid))
```

Comment:
Is it really the case that we're running into *boundary* variables that are larger than 4 GB each? I admit to being rather unfamiliar with what exactly is in boundary files.